plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetWriterConfig.java

```diff
@@ -67,11 +73,20 @@ public ParquetWriterConfig setParquetOptimizedWriterEnabled(boolean parquetOptim
         return this;
     }
 
+    @Config("parquet.default-bloom-filter-fpp")
```

Comment:
Why "default"? Can it be overridden in any way?

Reply:
I'd imagine that we would still want an end user be able to enable writing bloom filters even if they don't specify the fpp? The Orc configuration options also specifies fpp as a default property:

trino/plugin/trino-hive/src/main/java/io/trino/plugin/hive/orc/OrcWriterConfig.java

(For now this makes no difference since Parquet does not allow for fpp tuning so we could just omit this for now?)

Reply:
Right, but in that case I would just name the option simply `parquet.bloom-filter-fpp`. "default" would only make sense if it's truly a default that, for instance, can be overridden per query.